tests/ensemble_test.py

```diff
@@ -71,21 +72,7 @@ def test_c2st_posterior_ensemble_on_linearGaussian(
     )
 
     map_ = posterior.map(num_init_samples=1_000, show_progress_bars=False)
-
```

Comment:
Why remove this? It is fast and it checks (implicitly) whether `log_prob()` is doing the right thing

Reply:
yes, and it depends on the explicit value of `max_dkl` which appears to change when new (unrelated) tests are added. I thought having the `c2st` tests is enough.

Reply:
So the tests are quite stochastic given the random seed?

I'd prefer keeping the dkl test. `c2st` only checks the `.sample()` method. The `.log_prob()` of the `EnsemblePosterior` is not so trivial, so I would really prefer if this was covered by our tests

Reply:
they should not be stochastic, but they were failing on my branch even though I only refactored the `z-scoring-tests`. I increased the simulation budget to make the `SNRE` `c2st` check pass again.

Reply:
the `dkl` tests only apply to SNPE, right?

Reply:
I also works for single-round SNRE because it can also compute the log-prob (normalized)